Network/iTermSshMenu.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# <bitbar.title>iTerm SSH Dropdown Menu</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Gerd Naschenweng</bitbar.author>
+# <bitbar.author.github>magicdude4eva</bitbar.author.github>
+# <bitbar.desc>Provides a simple menu of ssh hosts when clicked on opens a new ssh session in terminal</bitbar.desc>
+# <bitbar.image></bitbar.image>
```

Comment:
Please take a screenshot of it running and include an image. It makes the plugin more likely to be seen and used.

Reply:
I hope this is fine:
![image](https://user-images.githubusercontent.com/1632781/107773097-70554080-6d3d-11eb-8599-e8f66776ae36.png)


And to see it running:
![iTermDropdown](https://user-images.githubusercontent.com/1632781/107773040-59165300-6d3d-11eb-85f4-16c4551a8795.gif)